src/app/js/fields/editFieldValue/FieldInput.js

```diff
@@ -53,6 +54,10 @@ export const FieldInputComponent = ({
             component={Component}
             disabled={field.name === 'uri'}
             label={label}
+            field={field}
+            validate={v =>
+                predicate(v) ? undefined : polyglot.t('bad_format_details')
```

Comment:
You're declaring this function each time you render this component.
Can you define an handler above please?